modules/edit-modes/src/lib/rotate-mode.js

```diff
@@ -1,84 +1,195 @@
 // @flow
 
+import turfBuffer from '@turf/buffer';
+import bbox from '@turf/bbox';
 import turfCentroid from '@turf/centroid';
 import turfBearing from '@turf/bearing';
+import bboxPolygon from '@turf/bbox-polygon';
+import turfDistance from '@turf/distance';
+import { coordEach } from '@turf/meta';
+import { getGeom } from '@turf/invariant';
+import { point, featureCollection, lineString } from '@turf/helpers';
 import turfTransformRotate from '@turf/transform-rotate';
 import type {
   PointerMoveEvent,
   StartDraggingEvent,
   StopDraggingEvent,
   DraggingEvent,
-  ModeProps
+  ModeProps,
+  EditHandleFeature
 } from '../types.js';
+import { getPickedEditHandle } from '../utils';
 import type { FeatureCollection, Position } from '../geojson-types.js';
-import { BaseGeoJsonEditMode, type GeoJsonEditAction } from './geojson-edit-mode.js';
+import {
+  BaseGeoJsonEditMode,
+  type GeoJsonEditAction,
+  getIntermediatePosition
+} from './geojson-edit-mode.js';
 import { ImmutableFeatureCollection } from './immutable-feature-collection.js';
 
 export class RotateMode extends BaseGeoJsonEditMode {
```

Comment:
Can you also add tests for rotate mode? We had tests before at `modules/layers/test/lib/mode-handlers/rotate-handler.test.js` that need ported over, but they probably need mostly rewritten since this mode works differently now.